docs/storefront/headless/channels.mdx

```diff
@@ -33,7 +33,7 @@ To tell BigCommerce the URL of your headless store, use the [Create a channel si
 
 ### Add a checkout URL
 
-Currently, all checkout pages are hosted. Set the checkout domain for the headless storefront by sending a request to the [Upsert a site's checkout URL](/docs/rest-management/channels/site-checkout-url#upsert-a-siteʼs-checkout-url) endpoint.
+Currently, all checkout pages are hosted. Set the checkout domain for the headless storefront by sending a request to the [Upsert a site's checkout URL](/docs/rest-management/channels/site-checkout-url#upsert-a-siteʼs-checkout-url) endpoint. You can configure the headless checkout url with `www_redirect` settings using the channels API.
```

Comment:
it's Settings API, not channels API. 
https://developer.bigcommerce.com/docs/rest-management/settings/storefront-seo#update-storefront-seo-settings